components/avatar/avatar.vue

```diff
@@ -85,6 +112,35 @@ export default {
     };
   },
 
+  computed: {
+    presenceStyles () {
+      // Adjust the position of presence
+      // based on the Avatar's size.
+      switch (this.size) {
+        case 'lg':
+          return {
+            bottom: '1px',
+            right: '1px',
+          };
+        case 'md':
+          return {
+            bottom: '-1px',
+            right: '-1px',
+          };
+        case 'sm':
+          return {
+            bottom: '-2px',
+            right: '-2px',
+          };
+        default:
+          return {
+            bottom: '0',
+            right: '0',
+          };
+      }
+    },
```

Comment:
What we probably should have done here is include these styles as classes within avatar in the dialtone repo, and then apply these classes based on the avatar size. This would align with our practices for most of our other components, and the classes would be documented in dialtone instead of hidden in the vue component

I'm okay with merging this as is though and then further iterating to make this change.

Reply:
Np, I think I'm fine with holding this PR until making that change in DT. I've seen that pattern before so I know what you mean - I created a draft [here](https://github.com/dialpad/dialtone/pull/739) based on that. Could you take a look and let me know? I'm not sure if we have a naming standard for targeting DT components within other DT components.

https://github.com/dialpad/dialtone/pull/739

When that's merged we'll just use a constant to map to these classes for presence here.

Reply:
I'll do a quick PR in `dialtone` to add it to https://dialpad.design/components/avatar.html

Reply:
@modialpad Sorry I got ahead of it and didn't notice your draft PR. What I did at https://github.com/dialpad/dialtone/pull/740 should do it.

Reply:
https://github.com/dialpad/dialtone/pull/740 has been merged to staging. I'll defer to Brad on timing of publishing so you can update this PR to use it.

Reply:
Awesome thanks!

Reply:
@braddialpad I updated this w/ @francisrupert's latest changes - let me know if it LG and I'll just merge.

One thing I noticed is that for the presence class modifiers we don't have an explicit modifier for `sm` case. It's actually the default, so it still works as expected, but in the tests I just had to assume that the styles are correct for the default case.